### PR TITLE
Add model for auditing user activity

### PR DESCRIPTION
**Pull request by fivetran-jamie**
## PR Overview
**This PR will address the following Issue/Feature:**
https://github.com/fivetran/dbt_fivetran_log/issues/89

**This PR will result in the following new package version:**

v1.3.0, as this will be batched with breaking change updates

**Please detail what change(s) this PR introduces and any additional information that should be known during the review of this PR:**

This creates a new model, `fivetran_platform__user_audit` (how do we feel about this name? i also considered audit_user_activity/user_activity_audit) to track user-triggered actions in fivetran connectors and destinations

## New Model Alert 👶🏽
[PR #98](https://github.com/fivetran/dbt_fivetran_log/pull/98) includes the following update:
- We have added a new model, `fivetran_platform__user_audit` ([source](https://fivetran.github.io/dbt_fivetran_log/#!/model/model.fivetran_log.fivetran_platform__user_audit)):
  - Each record represents a user-triggered action in your Fivetran instance. This model is intended for audit-trail purposes, as it can be very helpful when trying to trace a user action to a [log event](https://fivetran.com/docs/logs#logeventlist) such as a schema change, sync frequency update, manual update, broken connection, etc.
  - This model builds off of this [sample query](https://fivetran.com/docs/logs/fivetran-platform/sample-queries#audituseractionswithinconnector) from Fivetran's docs.

### Basic Validation
Please acknowledge that you have successfully performed the following commands locally:
- [x] dbt compile
- [x] dbt run –full-refresh
- [x] dbt run
- [x] dbt test
- [x] dbt run –vars (if applicable) tested out user/destination_membership variable configs

Before marking this PR as "ready for review" the following have been applied:
- [x] The appropriate issue has been linked and tagged
- [x] You are assigned to the corresponding issue and this PR
- [x] BuildKite integration tests are passing

### Detailed Validation
Please acknowledge that the following validation checks have been performed prior to marking this PR as "ready for review":
- [x] You have validated these changes and assure this PR will address the respective Issue/Feature.
- [x] You are reasonably confident these changes will not impact any other components of this package or any dependent packages.
- [x] You have provided details below around the validation steps performed to gain confidence in these changes.

See Hex notebook linked in height task

### Standard Updates
Please acknowledge that your PR contains the following standard updates:
- Package versioning has been appropriately indexed in the following locations:
    - [x] indexed within dbt_project.yml
    - [x] indexed within integration_tests/dbt_project.yml
- [x] CHANGELOG has individual entries for each respective change in this PR
    
- [x] README updates have been applied (if applicable)
    
- [ ] DECISIONLOG updates have been updated (if applicable)
- [x] Appropriate yml documentation has been added (if applicable)

### dbt Docs
Please acknowledge that after the above were all completed the below were applied to your branch:
- [ ] docs were regenerated (unless this PR does not include any code or yml updates) 

waiting to do so on release branch

### If you had to summarize this PR in an emoji, which would it be?
 
🌚

### CODE REVIEW

File: models/staging/stg_fivetran_platform.yml
Comment (fivetran-jamie) [inline models/staging/stg_fivetran_platform.yml:115]: this was fixed since the package was updated

File: CHANGELOG.md
Comment (fivetran-joemarkiewicz) [inline CHANGELOG.md:5]: Responding to your comment on the name of the new model. I actually really like your suggestion of `fivetran_platform__ audit_user_activity`. It matches with the naming convention of `fivetran_platform__audit_table` and accurately represents what one would find in the new table.

Comment (fivetran-joemarkiewicz) [reply CHANGELOG.md:5]: Can we make this update throughout the PR.

File: models/fivetran_platform__user_audit.sql
Comment (fivetran-joemarkiewicz) [inline models/fivetran_platform__user_audit.sql:14]: Can you explain this and statement? Is this to filter our fivetran system events?

File: packages.yml
Comment (fivetran-joemarkiewicz) [inline packages.yml:9]: Let's expand this to be `<1.0.0`. Similar to what we did in the [latest Zendesk update](https://github.com/fivetran/dbt_zendesk/blob/62df5b5fee7e448357f92c007c8212a0d6ece2f7/packages.yml#L6). We confirmed with Calogica that this would be appropriate and helps to reduce the dependency conflicts some may face in the future.

**Review by fivetran-joemarkiewicz — CHANGES_REQUESTED** [review]
This looks great and I really appreciate your validations that you shared to confirm these updates were appropriate! This will be a great end model to include in the next release of the package! I do have a few comments that I would like to be applied to this PR. Nothing too major, but enough that I will hold off approving until once the updates are applied.

Let me know if you have any questions!

File: models/fivetran_platform__user_audit.sql
Comment (fivetran-jamie) [reply models/fivetran_platform__user_audit.sql:14]: yeah so there are some fivetran-triggered events where the `actor` is just `Fivetran` (many other Fivetran events simply don't have an `actor` attribute). in our own test data, these Fivetran-actor log events are all like this
image


since the purpose of this model is audit user activity, i figured actions that are not actually attached to users should get the boot, but very open to opinions

Comment (fivetran-joemarkiewicz) [reply models/fivetran_platform__user_audit.sql]: That's what I assumed. Thanks for clarifying and I agree we should filter those out as I don't imagine users would be too interested in Fivetran actor events.

**Review by fivetran-joemarkiewicz — APPROVED** [review]
LGTM!

**Pull request merged**
